client/Ratings.jsx

```diff
@@ -73,38 +50,42 @@ export default class Ratings extends React.Component {
       accuracyAvg, communicationAvg, cleanlinessAvg,
       locationAvg, checkinAvg, valueAvg
     } = this.state;
-    const stars = Ratings.intToStars(4);
-    console.log(stars);
-
+    const { intToStars } = this.props;
     return (
       <div>
         <RatingsContainer>
           <RatingsLeftContainer>
-            Accuracy
-            {' '}
-            {Ratings.intToStars(accuracyAvg)}
-            <br />
-            Communication
-            {' '}
-            {Ratings.intToStars(communicationAvg)}
-            <br />
-            Cleanliness
-            {' '}
-            {Ratings.intToStars(cleanlinessAvg)}
-            <br />
+            <RatingTextAndStarsContainer>
```

Comment:
seems like you're repeating <RatingTextAndStarsContainer> a lot, any way to refactor? just a thought